src/number_theory/bernoulli.lean

```diff
@@ -139,8 +161,49 @@ begin
   rw ← cast_sub hk,
   congr',
   field_simp [show ((n - k : ℕ) : ℚ) + 1 ≠ 0, by {norm_cast, simp}],
-  -- down to nat
   norm_cast,
   rw [mul_comm, nat.sub_add_eq_add_sub hk],
   exact choose_mul_succ_eq n k,
 end
+
+open power_series
+open nat
+
+@[simp] lemma constant_coeff_exp : constant_coeff ℚ (exp ℚ) = 1 := rfl
```

Comment:
This lemma belongs in `ring_theory/power_series/well_known` next to `power_series.coeff_exp`.

Reply:
All done, thank you!